src/script/page/AppMain.tsx

```diff
@@ -74,6 +75,19 @@ const AppMain: FC<AppMainProps> = ({
 }) => {
   const apiContext = app.getAPIContext();
 
+  useInitializeRootFontSize();
+
+  useEffect(() => {
+    PrimaryModal.init();
+    showInitialModal(userAvailability);
+    // userAvailability not needed for dependency
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useLayoutEffect(() => {
+    initializeApp();
+  }, []);
```

Comment:
I think this look a little weird, we consume `initializeApp` before it was defined. 
Let's re-move this back to where is was initially